### PR TITLE
Fix "gpg" usage to stop relying on deprecated and insecure behavior

### PR DESCRIPTION
**Pull request by tianon**
See also docker-library/official-images#1420.

### CODE REVIEW

**Comment by tianon** [comment]
@jfrazelle pretty simple change; LGTY? :innocent:

File: Dockerfile [hunk 1]
```diff
@@ -40,10 +37,14 @@ RUN buildDeps=' \
 	&& rm -rf /var/lib/apt/lists/* \
 	&& curl -fsSL "https://github.com/irssi-import/irssi/releases/download/${IRSSI_VERSION}/irssi-${IRSSI_VERSION}.tar.bz2" -o /tmp/irssi.tar.bz2 \
 	&& curl -fsSL "https://github.com/irssi-import/irssi/releases/download/${IRSSI_VERSION}/irssi-${IRSSI_VERSION}.tar.bz2.sig" -o /tmp/irssi.tar.bz2.sig \
-	&& gpg --verify /tmp/irssi.tar.bz2.sig \
+	&& export GNUPGHOME="$(mktemp -d)" \
+# gpg: key DDBEF0E1: public key "The Irssi project <staff@irssi.org>" imported
+	&& gpg --keyserver pool.sks-keyservers.net --recv-keys 7EE65E3082A5FB06AC7C368D00CCB587DDBEF0E1 \
```
Comment (tianon) [inline Dockerfile]: Oh bollocks, I should update this to be `ha.` as well.

Comment (tianon) [inline Dockerfile]: Updated!

**Comment by jessfraz** [comment]
LGTM! Sorry! So behind!

**Comment by tianon** [comment]
No need to apologize; you're a hero :+1: :heart:

**Pull request merged**